docs/management/admin/endpoint-protection-rules.asciidoc

```diff
@@ -0,0 +1,37 @@
+[[endpoint-protection-rules]]
+= Endpoint protection rules
+
+Endpoint protection rules are prebuilt rules designed to help you manage and respond to alerts generated by {elastic-endpoint}, the installed component that performs {elastic-defend}'s threat monitoring and prevention. These rules include the <<endpoint-security, {elastic-defend}>> rule as well as additional detection and prevention rules for different {elastic-defend} protection features.
```

Comment:
I'm sure the team that generates the prebuilt rule docs is already on this, but in case they accidentally missed it (or we're naming the rule differently in the UI), I wanted to point out that the [docs](https://security-docs_bk_6100.docs-preview.app.elstc.co/guide/en/security/master/endpoint-security.html) for the Elastic Defend rule are still titled _Endpoint Security_.

Reply:
Hey @approksiu, do you know if/when the TRADE team will be updating the prebuilt rule docs for the Elastic Defend rule? Should we wait until those docs are updated before we release this?